src/mongodb_store.rs

```diff
@@ -64,6 +69,27 @@ impl MongoDBStore {
     fn col(&self) -> Collection<MongoDBSessionRecord> {
         self.client.database(&self.database).collection("sessions")
     }
+
+    /// Seperated function to setup the index for the session store.
+    /// This function needs to be called in order to let MongoDB delete
+    /// expired sessions internally.
+    pub async fn setup(&self) -> Result<(), MongoDBStoreError> {
```

Comment:
For SQLx stores we have a migrate method. This seems similar but perhaps a bit different in that it's adding an index rather than creating the schema. If setup is a familiar name for MongoDB folks then this seems reasonable.

Reply:
Actually it was ``create_index`` but I changed to ``setup`` because it sounds familiar to ``migration`` but since its not sql.

Reply:
It can be other name that sounds more meaning and useful than ``setup``

Reply:
I'll leave it up to you--I'm not familiar with the Mongo conventions. 😁

Reply:
I reckon ``setup`` is still appropriate for now.